purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -36,7 +36,7 @@ open class PurchasesConfiguration(builder: Builder) {
     val pendingTransactionsForPrepaidPlansEnabled: Boolean
 
     init {
-        this.context = builder.context
+        this.context = builder.context.applicationContext
```

Comment:
I noticed we could potentially be leaking an activity if the developer gave us the activity context and we hold a reference to this object. So I changed it here so we only hold the application context, which is the one developers should normally give us.